api/service/store/add.js

```diff
@@ -28,22 +28,25 @@ export function storeAddProvider(context) {
         )
       }
 
-      // Only use capability account for now to check if account is registered.
-      // This must change to access account/info!!
-      // We need to use https://github.com/web3-storage/w3protocol/blob/9d4b5bec1f0e870233b071ecb1c7a1e09189624b/packages/access/src/agent.js#L270
-      const account = capability.with
 
+      const resource = Server.DID.parse(capability.with).did()
       const [
+        verified,
         carIsLinkedToAccount,
         carExists
       ] = await Promise.all([
-        context.storeTable.exists(account, link.toString()),
+        context.access.verifyInvocation(invocation),
```

Comment:
do `context.access.verifyInvocation` before we do any other work? db and bucket reads have a smol $ cost.

Reply:
IDK, I'm optimistically hoping that requests are going to have access to do this.